integration/containerd/cri/integration-tests.sh

```diff
@@ -55,11 +55,12 @@ readonly CONTAINERD_CONFIG_FILE="${tmp_dir}/test-containerd-config"
 readonly default_containerd_config="/etc/containerd/config.toml"
 readonly default_containerd_config_backup="$CONTAINERD_CONFIG_FILE.backup"
 readonly kata_config="/etc/kata-containers/configuration.toml"
+readonly kata_config_backup="$kata_config.backup"
```

Comment:
It might be safer to do something like this:

```suggestion
readonly kata_config_backup=$(mktemp -p $(dirname "$kata_config") $(basename "$kata_config").XXXXXXXXXX)
```

Reply:
@jodh-intel looking at https://github.com/kata-containers/tests/pull/4429#discussion_r805679917, is it acceptable to use a static `.backup` or `.bak` suffix for the backed up files? I can't think of a non-fragile way to store and re-map the temps.

Reply:
@jodh-intel ping :)

Reply:
Looking at this again, this seems fine given we're already using the same strategy for containerd (although using a temporary directory there).